src/model/core/EthqlTransaction.ts

```diff
@@ -59,6 +59,17 @@ class EthqlTransaction {
   public async block(_, context: EthqlContext, info: GraphQLResolveInfo) {
     return this.blockNumber ? EthqlBlock.load(this.blockNumber, context, info) : null;
   }
+
+  public async status(_, context: EthqlContext) {
```

Comment:
Since we're only interested in the status field, we could use decomposition and a ternary expression:

```
const { status } = await context.web3.eth.getTransactionReceipt(this.hash);
return status === undefined ? null : status ? 'SUCCESS' : 'FALSE';
```

Reply:
Will change, thanks for the input